### PR TITLE
enh: update jbake dependencies to be able to use with JDK 11-21+ on I…

### PR DESCRIPTION
**Pull request by lprimak**
…ntel and Arm architectures

### CODE REVIEW

**Review by bmarwell — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>org.apache</groupId>
     <artifactId>apache</artifactId>
-    <version>24</version>
+    <version>31</version>
   </parent>
 
   <groupId>org.apache.shirp</groupId>
```
Comment (fpapon) [inline pom.xml:12]: Why do we have "shirp" rather than "shiro" in the `groupId`?

File: pom.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>org.apache</groupId>
     <artifactId>apache</artifactId>
-    <version>24</version>
+    <version>31</version>
```
Comment (fpapon) [inline pom.xml:9]: Is the upgrade of ASF pom parent mandatory for this PR? (I know that is good to upgrade but may be in another PR for the record and commit diff)

**Review by fpapon — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (fpapon) [reply pom.xml:12]: It seems to be an old error :(

Comment (bmarwell) [reply pom.xml:12]: Oops! Luckily just the site.

File: pom.xml (hunk 2 above)
Comment (lprimak) [reply pom.xml:9]: These need to go together. Separately things will break

File: pom.xml (hunk 1 above)
Comment (lprimak) [reply pom.xml:12]: this is now fixed

**Pull request merged**